Add CONTRIBUTING.md (#406)
mui-org/material-ui-x: CONTRIBUTING.md

# Contributing to Material-UI X

## Material-UI X vs. Material-UI

Material-UI X is an extension of Material-UI.

> The repositories are part of the same codebase.

Reviewer: What does this mean?

Author: Same problem, same solution

`mui-org/material-ui-x` imports the code infrastructure from [`mui-org/material-ui`](https://github.com/mui-org/material-ui).

> You can find the contribution guide of the main repository at [mui-org/material-ui/CONTRIBUTING.md](https://github.com/mui-org/material-ui/blob/HEAD/CONTRIBUTING.md).

Reviewer: ```suggestion
You can find the "contributing" guide for the main repository in [mui-org/material-ui/CONTRIBUTING.md](https://github.com/mui-org/material-ui/blob/HEAD/CONTRIBUTING.md).
```

> Most of the content on the contribution guide of Material-UI applies to this repository.

Reviewer: ```suggestion
Most of the content in the Material-UI "contributing" guide applies to this repository.
```

> However, there are a few differences:

Reviewer: ```suggestion
There are, however,  a few differences:
```
